stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -26,10 +37,12 @@ DOCKER_IMAGE_TAG=${DOCKER_IMAGE_TAG:-$SDE_VERSION-$KERNEL_VERSION}
 docker run -it --privileged \
```

Comment:
I might be wrong, but I think the container should be removed on stop, or at least I don't see any references about restarting a stopped stratum container
```suggestion
docker run --rm -it --privileged \
```

Reply:
Good point. Added to all containers.